rdmo/management/assets/js/components/common/Filter.js

```diff
@@ -61,4 +61,23 @@ FilterSite.propTypes = {
   onChange: PropTypes.func
 }
 
-export { FilterString, FilterUriPrefix, FilterSite }
+const FilterEditor = ({ value, options, onChange }) => {
```

Comment:
Maybe we can use the same component as for `sites`., but with a prop for `allLabel`

Reply:
thanks, Ive added the `allLabel` prop!